README.md

```diff
@@ -1,24 +1,69 @@
-# README
+# school-library-app
+The Blog app will be a classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
 
-Things you may want to cover:
+- Technologies used:
+  > Ruby
+  > Gemfile
+  > IRB
+  > Rails
+  > PostgreSql
+  > Rspec
 
-* Ruby version
 
-* System dependencies
+## Getting Started
 
-* Configuration
+### Prerequisites
 
-* Database creation
+You need to have a browser and this tools installed:
 
-* Database initialization
+- Git
+- NodeJS
+- Ruby
+- Rails
+- PostgreSql
 
-* How to run the test suite
+### Setup
 
-* Services (job queues, cache servers, search engines, etc.)
+To get a local copy up and running follow these simple example steps:
 
-* Deployment instructions
+```
+$ git clone git@github.com:tylher/school-library-app.git
+```
 
-* ...
+```
+$ cd school-library-app
+```
+
+```
+$ bundle install
+```
+
+### Usage
+
+To run any ruby file:
+
+```
+$ ruby (name of file)
+```
+
+To run app on server:
+```
+$ rails s
+```
+
+
+## Authors
+
+👤 <a href="https://github.com/tylher" target="blank"><img align="center"
+      src="https://user-images.githubusercontent.com/57408419/163676887-390d6032-6720-42bb-ad16-f8e199d6f2fa.jpg"
+      alt="adam pithewan" height="50" width="50"/></a> **Taiwo Damola Adediran**
+
+- GitHub: [@tylher](https://github.com/tylher)
+- Twitter: [@tylher123](https://twitter.com/tylher123)
+- LinkedIn: [Taiwo Adediran](https://www.linkedin.com/in/taiwo-adediran-327654127/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] [OPTIONAL] I think you forgot to put MIT license to your project. Kindly add this as well.